README.md

```diff
@@ -60,6 +60,41 @@ Experimenter is a platform for managing experiments in [Mozilla Firefox](https:/
 
 Done!
 
+### Semi Dockerized Setup
+
+[osx catalina, reinstall command line tools](https://medium.com/flawless-app-stories/gyp-no-xcode-or-clt-version-detected-macos-catalina-anansewaa-38b536389e8d)
+
+1.  Pre reqs (macOs instructions)
+
+        brew install postgresql llvm openssl yarn
+
+        echo 'export PATH="/usr/local/opt/llvm/bin:$PATH"' >> ~/.bash_profile
+        export LIBRARY_PATH=$LIBRARY_PATH:/usr/local/opt/openssl/lib/
+
+2.  Install dependencies
+
+        poetry install
+
+        yarn install
+
+3.  env values
+
+        .env:
+        DEBUG=True
+        POSTGRES_HOST=localhost
+        HOSTNAME=localhost
+
+4.  Start postgresql, redis, autograph, kinto
+
+        make up_db
```

Comment:
I got this at this step:

![Screen Shot 2020-09-15 at 11 43 16 AM](https://user-images.githubusercontent.com/6392049/93233082-a9669600-f748-11ea-9d58-a9eb9cb88180.png)

Reply:
ooh i've seen this with a fully dockerized set up as well, not sure how to handle that... maybe @jaredlockhart would know or could provide more insight?

Reply:
Yeah so the way we have kinto setup right now it's using an in memory db so when you restart it it starts over from scratch.  So if you start everything up, create a rapid experiment, push it to kinto and approve it, then stop all your containers, start them back up again, then the experiment will still be in experimenter but not in kinto.  

I've only used the kinto part to validate that the push works correctly so it hasn't caused me any trouble yet, but if it causes you in trouble in theory we can tell kinto to point to postgres, but we'd have to add a couple more steps to the setup to create all its tables etc, I took a stab at it recently and it was more than trivial so I didn't finish it.  

But if you find that you keep losing data you want in kinto and it gets in your way we can put some more time into it.

Reply:
Ah okay I was confused because it looked like it was hanging here (I assumed it would background these processes). So I can continue and ignore this

Reply:
No with `make up_db` it foregrounds them and they stay alive in that terminal, so I usually start them and then open a second terminal to do other stuff in.  If you want we can add a `make up_db_detached` that'll start them all in the background but you'll lose their output if you ever want to go back and check it for something.

Reply:
And yeah exactly you can just continue and ignore it, that all looks like it's running healthily to me.

Reply:
>  If you want we can add a make up_db_detached that'll start them all in the background

Meh, I'm indifferent. Foreground is good with me.